delfin/api/schemas/access_info.py

```diff
@@ -39,12 +39,11 @@
                              'maxLength': 255},
                 'password': {'type': 'string', 'minLength': 1,
                              'maxLength': 255},
-                'host_key': {'type': 'string', 'minLength': 1,
-                             'maxLength': 255},
-                'host_key_type': parameter_types.host_key_type
+                'pub_key': {'type': 'string', 'minLength': 1,
+                            'maxLength': 255},
```

Comment:
255 is too short for public key

Reply:
How about 1024 bytes ? Currently 2048 bits is considered reasonable for an RSA key; 4096 bits is higher than necessary but usually supported; having a length of 512 bytes is enough for a 4096 bits.. 1024 is a safe bet.. what do you think?